akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -28,95 +28,92 @@ akka.cluster.sharding {
   # Default is ddata for backwards compatibility.
   remember-entities-store = "ddata"
 
-  # Deprecated: use the `passivation.idle.timeout` for 'idle' `passivation.strategy` instead.
+  # Deprecated: use the `passivation.default.idle-entity.timeout` setting instead.
```

Comment:
isn't `default` wrong? `default-idle-strategy` or `strategy-defaults` ?

Reply:
Thanks, that's wrong. I did just name it `default` at first. Should be `default-idle-strategy` now.

Reply:
Fixed in https://github.com/akka/akka/pull/30968/commits/bca7e3545b9573f3652a5ff37273abe6102db609